appcenter-data/Data.js

```diff
@@ -25,9 +27,26 @@ const Data = {
     list,
     create,
     remove,
-    replace
+    replace,
+    setListener
 };
 
+/**
+ * Set the remote operation listener.
+ *
+ * @param {object} listenerMap - listener.
+ * @return Future asynchronous operation with result being true if enabled, false otherwise.
+ */
+function setListener(listenerMap) {
```

Comment:
Please pull the latest change from mbaas-data branch, I believe `setRemoteOperationListener` has already been added to this file.